src/execution_plan/ops/op_unwind.c

```diff
@@ -117,4 +117,3 @@ static void UnwindFree(OpBase *ctx) {
 		op->currentRecord = NULL;
 	}
 }
-
```

Comment:
What is our style policy on trailing newlines? My autoformatter is adding them, should I change that?

Reply:
A single newline at the end of each file